website/content/en/docs/advanced-topics/heterogeneous.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Heterogeneous Cluster Support
+linkTitle: Heterogeneous Cluster Support
+weight: 80
+---
```

Comment:
We have the doc https://sdk.operatorframework.io/docs/advanced-topics/multi-arch/ so wdyt about we do the changes on top of this one instead of creating a new doc?